requirements-dev.txt

```diff
@@ -1,9 +1,10 @@
 # TODO(sbc): switch to using Pipenv since it seems like that way to go
 # these day managing python deps.
-# These requirements are only needed for developers who want to run # flake8 on
-# the codebase, not for users of emscripten.
+# These requirements are only needed for developers who want to run flake8 on
+# the codebase and generate docs using Sphinx, not for users of emscripten.
 # Install with `pip3 install -r requirements-dev.txt`
 
 flake8==3.9.0
 flake8-unused-arguments==0.0.6
 coverage==5.5
+Sphinx==2.4.4
```

Comment:
Dies the package name really start with the capital letter like that?

Reply:
Yup, https://pypi.org/project/Sphinx/2.4.4/ and the sidebar on https://www.sphinx-doc.org/en/master/ use the capitalized version. It seems like the lowercase version also works, though.

Reply:
Lets keep them all lowercase here.  I think pip is not case sensitive but we have used a lowercase convention in this file.